src/parser/jobs/ast/modules/CelestialOpposition.tsx

```diff
@@ -47,27 +64,27 @@ export default class CelestialOpposition extends Module {
 		this.lastUse = event.timestamp
 	}
 
-	private onSect(event: BuffEvent) {
-		this.activeSect = getDataBy(this.data.statuses, 'id', event.ability.guid)
+	private onSect(event: Events['statusApply']) {
+		this.activeSect = getDataBy(this.data.statuses, 'id', event.status)
 	}
 
 	onComplete() {
 		const holdDuration = this.uses === 0 ? this.parser.currentDuration : this.totalHeld
-		const usesMissed = Math.floor(holdDuration / this.data.actions.CELESTIAL_OPPOSITION.cooldown)
-		const fightDuration = this.parser.fight.end_time - this.parser.fight.start_time
-		const maxUses = (fightDuration / this.data.actions.CELESTIAL_OPPOSITION.cooldown) - 1
+		const usesMissed = Math.floor(holdDuration / (this.data.actions.CELESTIAL_INTERSECTION.cooldown))
+		//TODO update max uses to not use the whole duration such as when there are no available targets
+		const maxUses = (this.parser.pull.duration / this.data.actions.CELESTIAL_INTERSECTION.cooldown) - 1
 
 		const WASTED_USE_TIERS = {
 			[maxUses * SEVERITY_MOD.MINOR]: SEVERITY.MINOR,
 			[maxUses * SEVERITY_MOD.MEDIUM]: SEVERITY.MEDIUM,
 			[maxUses * SEVERITY_MOD.MAJOR]: SEVERITY.MAJOR, // if not used at all, it'll be set to 100 for severity checking
```

Comment:
For this one, I agree that it needs changing; however, the target percentage is set at 50% for major
Since it's kind of arbitrary, I am just chalking it up to the //TODO portion since they relate to each other
I'll keep it in mind though since this comment is definitely going to be the same in another module that I left the same //TODO in based on a similar review note